src/test/java/com/google/sps/BigBusinessesTest.java

```diff
@@ -0,0 +1,152 @@
+
+package com.google.sps.data;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import java.util.List;
+import java.util.LinkedList;
+import java.util.ArrayList;
+import java.util.Iterator;
+import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertSame;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import static org.hamcrest.CoreMatchers.*;
+import com.google.maps.model.Photo;
+
+@RunWith(JUnit4.class)
+public final class BigBusinessesTest {
+  private MapLocation googlesOffice = new MapLocation(40.457177, -79.916696);
+  private List<Listing> sampleListOfBusinesses = new LinkedList<>();
+  private List<Listing> expectedListOfBusinesses = new LinkedList<>();  
+  private Listing[] sampleListingsOfBusinesses = {
+                new Listing("LA Fitness",
+                  "Address", 
+                  new MapLocation(40.457091, -79.915331), 
+                  3.9,
+                  null, 
+                  null),
+                new Listing("west elm", 
+                  "Address",
+                  new MapLocation(40.456279, -79.915015), 
+                  3.6, 
+                  null,
+                  null),
+                new Listing("McDonalds",
+                  "Address", 
+                  new MapLocation(40.459450, -79.918479), 
+                  2.6, 
+                  null,
+                  null),
+                new Listing("East End Brewing Company", 
+                  "Address",
+                  new MapLocation(40.459391, -79.911782), 
+                  4.7,
+                  null, 
+                  null),
+              new Listing("The Shiny Bean Coffee & Tea", 
+                  "Address",
+                  new MapLocation(40.496328, -79.944862), 
+                  4.9, 
+                  null,
+                  null),
+                new Listing("Weisshouse", 
+                  "Address",
+                  new MapLocation(40.456684, -79.925499), 
+                  4.3,
+                  null, 
+                  null)
+              };
+
+  private void populateListOfBusinessesIntoLinkedList(Listing[] arrayOfBusiness) {
+    for(Listing business: sampleListingsOfBusinesses){
+      sampleListOfBusinesses.add(business);
+    }
+  }
+  
```

Comment:
Looks like there's a typo here. As written, this re-adds business to the array that it's already in. 
Curious, why is a linked list specifically needed? Isn't the datatype here already an array?

Reply:
The reason I am re-adding the businesses from an array it already is, because their is no way to make a static linked list without using the .add method. This way if I want to make multiple different lists of businesses, I can make a static list and then populate it into a linked list easily. Also, this is what you said I should do in this comment, https://github.com/googleinterns/SmallCity/pull/16#discussion_r452552544

Reply:
Not quite. The resolution for that comment is the list on line 49. Let's chat over a call about this to clarify what's happening.